[Bug Skip]: [Regression]UT failures 2026-6-25

### 🐛 Describe the bug with skip template

Cases:
~~op_ut,third_party.torch-xpu-ops.test.xpu.dynamo.test_functions_xpu.FunctionTests,test_partials_graph_break_reconstruct_args_and_kwargs~~
~~op_ut,third_party.torch-xpu-ops.test.xpu.dynamo.test_functions_xpu.FunctionTests,test_partials_graph_break_reconstruct_mix~~
~~op_ut,third_party.torch-xpu-ops.test.xpu.dynamo.test_functions_xpu.FunctionTests,test_partials_graph_break_reconstruct_mix_no_source~~
op_ut,third_party.torch-xpu-ops.test.xpu.export.test_hop_xpu.TestHOPXPU,test_serialize_export_switch_simple_xpu_float32
~~op_ut,third_party.torch-xpu-ops.test.xpu.dynamo.test_ctx_manager_xpu.ContextlibContextManagerTests,test_ctx_basic0~~
~~op_ut,third_party.torch-xpu-ops.test.xpu.dynamo.test_ctx_manager_xpu.CtxManagerTests,test_disable_saved_tensors_hooks_prev_disabled~~
~~op_ut,third_party.torch-xpu-ops.test.xpu.dynamo.test_ctx_manager_xpu.CtxManagerTests,test_disable_saved_tensors_hooks_prev_disabled_nested~~
~~op_ut,third_party.torch-xpu-ops.test.xpu.dynamo.test_ctx_manager_xpu.CtxManagerTests,test_disable_saved_tensors_hooks~~
~~op_ut,third_party.torch-xpu-ops.test.xpu.dynamo.test_ctx_manager_xpu.CtxManagerTests,test_disable_saved_tensors_hooks_graph_break~~

## ErrorLog
### AssertionError: 'clas[160 chars]]" = mul * l_x_;  mul = None\n        mul_2: "[138 chars],)\n' != 'clas[160 chars]]" = 20 * l_x_\n        mul_2: "f32[2, 2]" = m[131 chars],)\n'
```bash
_____ FunctionTests.test_partials_graph_break_reconstruct_args_and_kwargs ______
[gw4] linux -- Python 3.10.20 /tmp/xpu-tool/Python/3.10.20/x64/bin/python
Traceback (most recent call last):
  File "/__w/torch-xpu-ops/torch-xpu-ops/pytorch/third_party/torch-xpu-ops/test/xpu/dynamo/test_functions_xpu.py", line 3341, in test_partials_graph_break_reconstruct_args_and_kwargs
    self.assertExpectedInline(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_dynamo/test_case.py", line 135, in assertExpectedInline
    return super().assertExpectedInline(*args, **kwargs)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 3533, in assertExpectedInline
    return super().assertExpectedInline(actual if isinstance(actual, str) else str(actual), expect, skip + 1)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 413, in assertExpectedInline
    assert_expected_inline(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 378, in assert_expected_inline
    assert_eq(expect, actual, msg=help_text)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 450, in assertMultiLineEqualMaybeCppStack
    self.assertMultiLineEqual(expect, actual, *args, **kwargs)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/unittest/case.py", line 1226, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/unittest/case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: 'clas[160 chars]]" = mul * l_x_;  mul = None\n        mul_2: "[138 chars],)\n' != 'clas[160 chars]]" = 20 * l_x_\n        mul_2: "f32[2, 2]" = m[131 chars],)\n'
  class GraphModule(torch.nn.Module):
      def forward(self, L_x_: "f32[2, 2]"):
          l_x_ = L_x_
  
          mul: "f32[2, 2]" = l_x_ * 4
+         mul_1: "f32[2, 2]" = 20 * l_x_
-         mul_1: "f32[2, 2]" = mul * l_x_;  mul = None
?             ^
+         mul_2: "f32[2, 2]" = mul * l_x_;  mul = l_x_ = None
?             ^                                  +++++++
-         mul_2: "f32[2, 2]" = 20 * l_x_;  l_x_ = None
  
-         mul_3: "f32[2, 2]" = torch.mul(mul_1, mul_2);  mul_1 = mul_2 = None
?                                             ^      ^^^        --------
+         mul_3: "f32[2, 2]" = torch.mul(mul_2, mul_1);  mul_2 = mul_1 = None
?                                            +++++++ ^^^      ^^
          return (mul_3,)
 : To accept the new output, re-run test with envvar EXPECTTEST_ACCEPT=1 (we recommend staging/committing your changes before doing this)

To execute this test, run the following from the base repo dir:
    PYTORCH_TEST_WITH_SLOW=1 python test/xpu/dynamo/test_functions_xpu.py FunctionTests.test_partials_graph_break_reconstruct_args_and_kwargs

This message can be suppressed by setting PYTORCH_PRINT_REPRO_ON_FAILURE=0
____________ FunctionTests.test_get_device_properties_tensor_device ____________
[gw1] linux -- Python 3.10.20 /tmp/xpu-tool/Python/3.10.20/x64/bin/python
Traceback (most recent call last):
  File "/__w/torch-xpu-ops/torch-xpu-ops/pytorch/third_party/torch-xpu-ops/test/xpu/dynamo/test_functions_xpu.py", line 115, in test_fn
    return torch._dynamo.testing.standard_test(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_dynamo/testing.py", line 403, in standard_test
    correct1 = fn(*args1)
  File "/__w/torch-xpu-ops/torch-xpu-ops/pytorch/third_party/torch-xpu-ops/test/xpu/dynamo/test_functions_xpu.py", line 1290, in test_get_device_properties_tensor_device
    return x + prop.max_threads_per_multi_processor
AttributeError: 'torch._C._XpuDeviceProperties' object has no attribute 'max_threads_per_multi_processor'
```

### NotImplementedError: The operator 'aten::linalg_polar.out' is not currently implemented for the XPU device.
```bash
__________________ TestLinalgXPU.test_polar_empty_xpu_float32 __________________
[gw7] linux -- Python 3.10.20 /tmp/xpu-tool/Python/3.10.20/x64/bin/python
Traceback (most recent call last):
  File "/__w/torch-xpu-ops/torch-xpu-ops/pytorch/third_party/torch-xpu-ops/test/xpu/../../../../test/test_linalg.py", line 4455, in test_polar_empty
    U, H = torch.linalg.polar(A)
NotImplementedError: The operator 'aten::linalg_polar.out' is not currently implemented for the XPU device. Please open a feature on https://github.com/intel/torch-xpu-ops/issues. You can set the environment variable `PYTORCH_ENABLE_XPU_FALLBACK=1` to use the CPU implementation as a fallback for XPU unimplemented operators. WARNING: this will bring unexpected performance compared with running natively on XPU.

To execute this test, run the following from the base repo dir:
    PYTORCH_TEST_WITH_SLOW=1 python ../../test/test_linalg.py TestLinalgXPU.test_polar_empty_xpu_float32

This message can be suppressed by setting PYTORCH_PRINT_REPRO_ON_FAILURE=0
__________________ TestLinalgXPU.test_polar_empty_xpu_float64 __________________
[gw7] linux -- Python 3.10.20 /tmp/xpu-tool/Python/3.10.20/x64/bin/python
Traceback (most recent call last):
  File "/__w/torch-xpu-ops/torch-xpu-ops/pytorch/third_party/torch-xpu-ops/test/xpu/../../../../test/test_linalg.py", line 4455, in test_polar_empty
    U, H = torch.linalg.polar(A)
NotImplementedError: The operator 'aten::linalg_polar.out' is not currently implemented for the XPU device. Please open a feature on https://github.com/intel/torch-xpu-ops/issues. You can set the environment variable `PYTORCH_ENABLE_XPU_FALLBACK=1` to use the CPU implementation as a fallback for XPU unimplemented operators. WARNING: this will bring unexpected performance compared with running natively on XPU.
```

## Pytorch Version
latest good : 206283a284889f3a4f62510f8b5a8068d41121c4
current : e683636150f35c3972bde18ac610291f3e9e577f 

### Versions

<details>
<summary>Detail</summary>
Collecting environment information...
PyTorch version: 2.14.0a0+gite683636
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: Ubuntu 24.04.4 LTS (x86_64)
GCC version: (Ubuntu 13.3.0-6ubuntu2~24.04.1) 13.3.0
Clang version: Could not collect
CMake version: version 3.31.6
Libc version: glibc-2.39

Python version: 3.10.20 (main, Jun 16 2026, 21:01:32) [GCC 13.3.0] (64-bit runtime)
Python platform: Linux-5.15.0-179-generic-x86_64-with-glibc2.39
Is CUDA available: False
CUDA runtime version: No CUDA
CUDA_MODULE_LOADING set to: N/A
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
Is XPU available: True
XPU used to build PyTorch: 20260000
Intel GPU driver version:
* libze1:	1.24.0.0-1146~24.04
* intel-opencl-icd:	25.18.33578.51-1146~24.04
Intel GPU models onboard:
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
Intel GPU models detected:
* [0] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1100', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBDA, uuid=8680da0b-2f00-0000-0f00-000000000000, driver_version='1.6.33578+51', total_memory=49136MB, local_mem_size=128KB, last_level_cache_size=196608KB, max_compute_units=448, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=448, gpu_subslice_count=56, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1, is_integrated_gpu=0)
* [1] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1100', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBDA, uuid=8680da0b-2f00-0000-1600-000000000000, driver_version='1.6.33578+51', total_memory=49136MB, local_mem_size=128KB, last_level_cache_size=196608KB, max_compute_units=448, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=448, gpu_subslice_count=56, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1, is_integrated_gpu=0)
Vulnerability Spec store bypass:         Mitigation; Speculative Store Bypass disabled via prctl and seccomp
Vulnerability Spectre v1:                Mitigation; usercopy/swapgs barriers and __user pointer sanitization
Vulnerability Spectre v2:                Mitigation; Enhanced / Automatic IBRS; IBPB conditional; PBRSB-eIBRS SW sequence; BHI BHI_DIS_S
Vulnerability Srbds:                     Not affected
Vulnerability Tsa:                       Not affected
Vulnerability Tsx async abort:           Not affected
Vulnerability Vmscape:                   Mitigation; IBPB before exit to userspace

Versions of relevant libraries:
[pip3] dpcpp-cpp-rt==2026.0.0
[pip3] impi-rt==2021.18.0
[pip3] intel-cmplr-lib-rt==2026.0.0
[pip3] intel-cmplr-lib-ur==2026.0.0
[pip3] intel-cmplr-lic-rt==2026.0.0
[pip3] intel-opencl-rt==2026.0.0
[pip3] intel-openmp==2026.0.0
[pip3] intel-pti==0.17.0
[pip3] intel-sycl-rt==2026.0.0
[pip3] mkl==2026.0.0
[pip3] mypy==1.16.0
[pip3] mypy_extensions==1.1.0
[pip3] numpy==1.24.4
[pip3] nvidia-cuda-cupti==13.3.35
[pip3] oneccl==2022.0.0
[pip3] oneccl-devel==2022.0.0
[pip3] onemkl-license==2026.0.0
[pip3] onemkl-sycl-blas==2026.0.0
[pip3] onemkl-sycl-dft==2026.0.0
[pip3] onemkl-sycl-lapack==2026.0.0
[pip3] onemkl-sycl-rng==2026.0.0
[pip3] onemkl-sycl-sparse==2026.0.0
[pip3] onnx==1.21.0
[pip3] onnx-ir==0.1.16
[pip3] onnxscript==0.6.2
[pip3] optree==0.13.0
[pip3] tbb==2023.0.0
[pip3] tcmlib==1.5.0
[pip3] torch==2.14.0a0+gite683636
[pip3] torchao==0.18.0.dev20260625+xpu
[pip3] torchaudio==2.11.0a0+c0cbdb9
[pip3] torchvision==0.28.0a0+85a1dcc
[pip3] triton-xpu==3.7.2+git5fcc14d9
[pip3] umf==1.1.0
[conda] No relevant packages
</details>

--- a/test/xpu/dynamo/test_ctx_manager_xpu.py
+++ b/test/xpu/dynamo/test_ctx_manager_xpu.py
@@ -1309,15 +1309,15 @@ class GraphModule(torch.nn.Module):
     def forward(self):
         _saved_tensors_hooks_disable = torch._C._autograd._saved_tensors_hooks_disable('This is not supported');  _saved_tensors_hooks_disable = None
 
-        x: "f32[1]" = torch.ones(1)
+        ones: "f32[1]" = torch.ones(1)
 
-        y: "f32[1]" = torch.zeros(1)
+        zeros: "f32[1]" = torch.zeros(1)
 
-        add: "f32[1]" = x + y;  x = y = None
+        add: "f32[1]" = ones + zeros;  ones = zeros = None
 
         _saved_tensors_hooks_enable = torch._C._autograd._saved_tensors_hooks_enable();  _saved_tensors_hooks_enable = None
         return (add,)
-""",  # NOQA: B950
+""",
         )
 
     def test_disable_saved_tensors_hooks_prev_disabled(self):
@@ -1352,15 +1352,15 @@ class GraphModule(torch.nn.Module):
     def forward(self):
         _saved_tensors_hooks_disable = torch._C._autograd._saved_tensors_hooks_disable('This is not supported');  _saved_tensors_hooks_disable = None
 
-        x: "f32[1]" = torch.ones(1)
+        ones: "f32[1]" = torch.ones(1)
 
-        y: "f32[1]" = torch.zeros(1)
+        zeros: "f32[1]" = torch.zeros(1)
 
-        add: "f32[1]" = x + y;  x = y = None
+        add: "f32[1]" = ones + zeros;  ones = zeros = None
 
         _saved_tensors_hooks_disable_1 = torch._C._autograd._saved_tensors_hooks_disable('Previously disabled message');  _saved_tensors_hooks_disable_1 = None
         return (add,)
-""",  # NOQA: B950
+""",
         )
 
     def test_disable_saved_tensors_hooks_prev_disabled_nested(self):
@@ -1401,21 +1401,21 @@ class GraphModule(torch.nn.Module):
     def forward(self):
         _saved_tensors_hooks_disable = torch._C._autograd._saved_tensors_hooks_disable('This is not supported');  _saved_tensors_hooks_disable = None
 
-        x: "f32[1]" = torch.ones(1)
+        ones: "f32[1]" = torch.ones(1)
 
-        y: "f32[1]" = torch.zeros(1)
+        zeros: "f32[1]" = torch.zeros(1)
 
         _saved_tensors_hooks_disable_1 = torch._C._autograd._saved_tensors_hooks_disable('This is not supported inner');  _saved_tensors_hooks_disable_1 = None
 
-        add: "f32[1]" = x + y;  y = None
+        add: "f32[1]" = ones + zeros;  zeros = None
 
         _saved_tensors_hooks_disable_2 = torch._C._autograd._saved_tensors_hooks_disable('This is not supported');  _saved_tensors_hooks_disable_2 = None
 
-        add_1: "f32[1]" = add + x;  add = x = None
+        add_1: "f32[1]" = add + ones;  add = ones = None
 
         _saved_tensors_hooks_disable_3 = torch._C._autograd._saved_tensors_hooks_disable('Previously disabled message');  _saved_tensors_hooks_disable_3 = None
         return (add_1,)
-""",  # NOQA: B950
+""",
         )
 
     def test_disable_saved_tensors_hooks_graph_break(self):
@@ -1430,7 +1430,7 @@ class GraphModule(torch.nn.Module):
         eager = EagerAndRecordGraphs()
         torch.compile(fn, backend=eager, fullgraph=False)(torch.randn(()))
 
-        def check_graph(actual, expected):  # noqa: F841
+        def check_graph(actual, expected):
             self.assertExpectedInline(actual, expected)
 
         graph = eager.graphs[0]
@@ -1444,11 +1444,11 @@ class GraphModule(torch.nn.Module):
 
         _saved_tensors_hooks_disable = torch._C._autograd._saved_tensors_hooks_disable('This is not supported');  _saved_tensors_hooks_disable = None
 
-        y: "f32[]" = l_x_ + 1;  l_x_ = None
+        add: "f32[]" = l_x_ + 1;  l_x_ = None
 
         _saved_tensors_hooks_enable = torch._C._autograd._saved_tensors_hooks_enable();  _saved_tensors_hooks_enable = None
-        return (y,)
-""",  # NOQA: B950
+        return (add,)
+""",
         )
 
         graph = eager.graphs[1]
@@ -1466,7 +1466,7 @@ class GraphModule(torch.nn.Module):
 
         _saved_tensors_hooks_enable = torch._C._autograd._saved_tensors_hooks_enable();  _saved_tensors_hooks_enable = None
         return (mul,)
-""",  # NOQA: B950
+""",
         )
 
     def test_context_wrapping_grad_mode_decorator(self):
@@ -1966,10 +1966,10 @@ class GraphModule(torch.nn.Module):
     def forward(self):
         set_default_dtype = torch.set_default_dtype(torch.float64);  set_default_dtype = None
 
-        x: "c128[2]" = torch.tensor([3.0, (3+5j)])
+        tensor: "c128[2]" = torch.tensor([3.0, (3+5j)])
 
         set_default_dtype_1 = torch.set_default_dtype(torch.float32);  set_default_dtype_1 = None
-        return (x,)
+        return (tensor,)
 """,
         )
 

--- a/test/xpu/dynamo/test_functions_xpu.py
+++ b/test/xpu/dynamo/test_functions_xpu.py
@@ -3214,7 +3214,7 @@ class GraphModule(torch.nn.Module):
                 normalize_gm(backend.graphs[0].print_readable(print_output=False)),
                 """\
 class GraphModule(torch.nn.Module):
-    def forward(self, s9: "Sym(s9)", L_lambda0_keywords_y_: "f32[s9, s9]"):
+    def forward(self, L_lambda0_keywords_y_: "f32[s9, s9]", s9: "Sym(s9)"):
         l_lambda0_keywords_y_ = L_lambda0_keywords_y_
 
         mul: "f32[s9, s9]" = l_lambda0_keywords_y_ * l_lambda0_keywords_y_
@@ -3248,9 +3248,9 @@ class GraphModule(torch.nn.Module):
     def forward(self, L_lambda0_keywords_y_: "f32[2, 2]"):
         l_lambda0_keywords_y_ = L_lambda0_keywords_y_
 
-        mul: "f32[2, 2]" = l_lambda0_keywords_y_ * l_lambda0_keywords_y_
+        add: "f32[2, 2]" = l_lambda0_keywords_y_ + l_lambda0_keywords_y_
 
-        add: "f32[2, 2]" = l_lambda0_keywords_y_ + l_lambda0_keywords_y_;  l_lambda0_keywords_y_ = None
+        mul: "f32[2, 2]" = l_lambda0_keywords_y_ * l_lambda0_keywords_y_;  l_lambda0_keywords_y_ = None
 
         mul_1: "f32[2, 2]" = torch.mul(mul, add);  mul = add = None
         return (mul_1,)
@@ -3261,12 +3261,12 @@ class GraphModule(torch.nn.Module):
                 normalize_gm(backend.graphs[0].print_readable(print_output=False)),
                 """\
 class GraphModule(torch.nn.Module):
-    def forward(self, s9: "Sym(s9)", L_lambda0_keywords_y_: "f32[s9, s9]"):
+    def forward(self, L_lambda0_keywords_y_: "f32[s9, s9]", s9: "Sym(s9)"):
         l_lambda0_keywords_y_ = L_lambda0_keywords_y_
 
-        mul: "f32[s9, s9]" = l_lambda0_keywords_y_ * l_lambda0_keywords_y_
+        add: "f32[s9, s9]" = l_lambda0_keywords_y_ + l_lambda0_keywords_y_
 
-        add: "f32[s9, s9]" = l_lambda0_keywords_y_ + l_lambda0_keywords_y_;  l_lambda0_keywords_y_ = None
+        mul: "f32[s9, s9]" = l_lambda0_keywords_y_ * l_lambda0_keywords_y_;  l_lambda0_keywords_y_ = None
 
         mul_1: "f32[s9, s9]" = torch.mul(mul, add);  mul = add = None
         return (mul_1,)
@@ -3298,9 +3298,9 @@ class GraphModule(torch.nn.Module):
     def forward(self, L_lambda0_keywords_y_: "f32[2, 2]"):
         l_lambda0_keywords_y_ = L_lambda0_keywords_y_
 
-        mul: "f32[2, 2]" = l_lambda0_keywords_y_ * l_lambda0_keywords_y_
+        add: "f32[2, 2]" = l_lambda0_keywords_y_ + l_lambda0_keywords_y_
 
-        add: "f32[2, 2]" = l_lambda0_keywords_y_ + l_lambda0_keywords_y_;  l_lambda0_keywords_y_ = None
+        mul: "f32[2, 2]" = l_lambda0_keywords_y_ * l_lambda0_keywords_y_;  l_lambda0_keywords_y_ = None
 
         mul_1: "f32[2, 2]" = torch.mul(mul, add);  mul = add = None
         return (mul_1,)
@@ -3311,12 +3311,12 @@ class GraphModule(torch.nn.Module):
                 normalize_gm(backend.graphs[0].print_readable(print_output=False)),
                 """\
 class GraphModule(torch.nn.Module):
-    def forward(self, s9: "Sym(s9)", L_lambda0_keywords_y_: "f32[s9, s9]"):
+    def forward(self, L_lambda0_keywords_y_: "f32[s9, s9]", s9: "Sym(s9)"):
         l_lambda0_keywords_y_ = L_lambda0_keywords_y_
 
-        mul: "f32[s9, s9]" = l_lambda0_keywords_y_ * l_lambda0_keywords_y_
+        add: "f32[s9, s9]" = l_lambda0_keywords_y_ + l_lambda0_keywords_y_
 
-        add: "f32[s9, s9]" = l_lambda0_keywords_y_ + l_lambda0_keywords_y_;  l_lambda0_keywords_y_ = None
+        mul: "f32[s9, s9]" = l_lambda0_keywords_y_ * l_lambda0_keywords_y_;  l_lambda0_keywords_y_ = None
 
         mul_1: "f32[s9, s9]" = torch.mul(mul, add);  mul = add = None
         return (mul_1,)
@@ -3346,10 +3346,10 @@ class GraphModule(torch.nn.Module):
         l_x_ = L_x_
 
         mul: "f32[2, 2]" = l_x_ * 4
-        mul_1: "f32[2, 2]" = mul * l_x_;  mul = None
-        mul_2: "f32[2, 2]" = 20 * l_x_;  l_x_ = None
+        mul_1: "f32[2, 2]" = 20 * l_x_
+        mul_2: "f32[2, 2]" = mul * l_x_;  mul = l_x_ = None
 
-        mul_3: "f32[2, 2]" = torch.mul(mul_1, mul_2);  mul_1 = mul_2 = None
+        mul_3: "f32[2, 2]" = torch.mul(mul_2, mul_1);  mul_2 = mul_1 = None
         return (mul_3,)
 """,
             )
@@ -3358,14 +3358,14 @@ class GraphModule(torch.nn.Module):
                 normalize_gm(backend.graphs[0].print_readable(print_output=False)),
                 """\
 class GraphModule(torch.nn.Module):
-    def forward(self, s77: "Sym(s77)", L_x_: "f32[s77, s77]"):
+    def forward(self, L_x_: "f32[s77, s77]", s77: "Sym(s77)"):
         l_x_ = L_x_
 
         mul: "f32[s77, s77]" = l_x_ * 4
-        mul_1: "f32[s77, s77]" = mul * l_x_;  mul = None
-        mul_2: "f32[s77, s77]" = 20 * l_x_;  l_x_ = None
+        mul_1: "f32[s77, s77]" = 20 * l_x_
+        mul_2: "f32[s77, s77]" = mul * l_x_;  mul = l_x_ = None
 
-        mul_3: "f32[s77, s77]" = torch.mul(mul_1, mul_2);  mul_1 = mul_2 = None
+        mul_3: "f32[s77, s77]" = torch.mul(mul_2, mul_1);  mul_2 = mul_1 = None
         return (mul_3,)
 """,
             )

--- a/test/xpu/export/test_hop_xpu.py
+++ b/test/xpu/export/test_hop_xpu.py
@@ -147,7 +147,11 @@ class TestHOP(TestCase):
         torchdynamo._reset_guarded_backend_cache()
 
     @ops(hop_tests, allowed_dtypes=(torch.float,))
-    @skipOps(hop_export_failures | inline_asm_rocm_serialize_skips)
+    @skipOps(
+        hop_export_failures
+        | inline_asm_rocm_serialize_skips
+        | {xfail("switch", "simple")}
+    )
     def test_serialize_export(self, device, dtype, op):
         class Foo(torch.nn.Module):
             def forward(self, *args):

